### PR TITLE
Fix: Project banners aren't showing in collections

### LINKED ISSUE #1558: Project banners aren't showing in collections
### Describe the bug

Project banners aren't showing on selected collection page

Project banners are showing in search:
Снимок экрана (753)

Project banners aren't showing in collections (only gradient based on the icon):
Снимок экрана (754)

### Steps to reproduce

1. Go to Collections page
2. Go to any collection (if you don't have any, create one)
3. Add project(s) with custom banners
4. You'll see that project(s) in collection don't show banners, but only gradient based on the icon

### Expected behavior

Project banners are showing in collections

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Mysterious-Dev**
All is in the title ^^.

Close #1558.

### CODE REVIEW

File: pages/collection/[id].vue [hunk 1]
```diff
@@ -284,6 +284,12 @@
             :description="project.description"
             :downloads="project.downloads ? project.downloads.toString() : '0'"
             :follows="project.followers ? project.followers.toString() : '0'"
+            :featured-image="
```
Comment (Prospector) [inline pages/collection/[id].vue:287]: It should just filter by featured and show nothing if none are featured, it's a backend bug that the first non-featured one is shown in search.

**Review by Prospector — CHANGES_REQUESTED** [review]

File: pages/collection/[id].vue (hunk 1 above)
Comment (Mysterious-Dev) [reply pages/collection/[id].vue:287]: So I just need to use find() function?

Comment (Prospector) [reply pages/collection/[id].vue:287]: yeah, to get the first gallery image that is featured. if none, then show none

Comment (Mysterious-Dev) [reply pages/collection/[id].vue:287]: Done (I think) ^^.

Comment (Mysterious-Dev) [reply pages/collection/[id].vue]: Now that's fixed ^^.

**Review by Prospector — APPROVED** [review]

**Pull request merged**